samples/subsys/ipc/openamp/sample.yaml

```diff
@@ -6,6 +6,11 @@ tests:
     filter: dt_chosen_enabled("zephyr,ipc") and dt_chosen_enabled("zephyr,ipc_shm")
     integration_platforms:
       - mps2/an521/cpu0
+    platform_exclude:
```

Comment:
Please, remove this. It's been handled in #87139 or #87165

Reply:
Oh nice! I was indeed suspecting that this problem was not only for `twatch_s3` board, but for all the boards based on the S3 chip. Ok, I will remove that commit and add those PRs as dependency for this PR

Reply:
@sylvioalves is there any update about #87139 or #87165? So far this is the last failure I have on the CI of this PR, but it seems to me that there has been no activity on those PRs since last week.